### ISSUE TITLE
Style/IdenticalConditionalBranches should be marked as unsafe

### ISSUE THREAD

**Issue by Drowze** [rubocop/rubocop#9980]
## Expected behavior

I expect safe autocorrection not to break my existing ruby code.

## Actual behavior

Running a cop marked as safe breaks my existing ruby code.

## Steps to reproduce the problem

Given `test.rb`
```ruby
x = 0

if x == 0
  x += 1
  puts "if"
else
  x += 1
  puts "else"
end

# output when executed:
# if
```

When I run `rubocop -a --only Style/IdenticalConditionalBranches`, it autocorrects my file to:
```ruby
x = 0

x += 1
if x == 0
  puts "if"
else
  puts "else"
end

# output when executed:
# else
```

## RuboCop version

```
$ rubocop -V
1.18.4 (using Parser 3.0.2.0, rubocop-ast 1.8.0, running on ruby 2.7.3 x86_64-darwin20)
```

**Comment by koic** [comment]
This is a false positive bug because this cop can safe detect the offense.
